cfme/configure/configuration/region_settings.py

```diff
@@ -955,7 +958,13 @@ def is_displayed(self):
 class ReplicationGlobalView(ReplicationView):
     """ Replication Global setup View"""
     add_subscription = Button('Add Subscription')
-    subscription_table = VanillaTable('//form[@id="form_div"]//table[contains(@class, "table")]')
+    subscription_table = VanillaTable(
+        '//form[@id="form_div"]//table[contains(@class, "table")]',
+        column_widgets={
+            "Actions": Button("Update"),
+            10: Kebab(locator='//td[10]/div[contains(@class, "dropdown-kebab-pf")]')
```

Comment:
Why `10`? I think we should give this a more meaningful name

Reply:
Um, no reason in particular, just that it was the 10th column that I wanted and I saw this in the piece of code I was referring. Changing `10` to `Kebab`